core/templates/pages/teach-page/teach-page.component.html

```diff
@@ -45,7 +45,7 @@ <h1 class="oppia-teach-h1" [innerHTML]="'I18N_TEACH_PAGE_HEADING' | translate"><
       <div *ngIf="androidPageIsEnabled" class="oppia-splash-button-container">
         <button type="submit"
                 (click)="onClickAccessAndroidButton()"
-                class="btn oppia-splash-button oppia-splash-button-browse oppia-splash-browse-button-position oppia-transition-200">
```

Comment:
Oppia-splash-button-browse class is used at some other place, if not then please remove its definition too.

Reply:
Done

Reply:
I am not able to see any changes in PR.

Reply:
i have changed oppia-splash-button-container to oppia-teach-button-container in line 45, what else needs to be done?

Reply:
does this class have any type of definition 
like this .oppia-splash-button-container{
margin: 28 px;
}

if yes does that definition is still required or we can remove this definition now.

Reply:
It has the definition in the splash page and is being used there. The code for the two buttons might have been copied from there but had to be changed from oppia-splash-button-container to oppia-teach-button-container which was not done in this specific button in the mobile view case which caused the error. It does not have a definition in the teach-page components and hence no definition has to changed.